Languages/IronPython/IronPython/Modules/sys.cs

```diff
@@ -177,7 +177,7 @@ public static class SysModule {
 
         public static object getfilesystemencoding() {
             if(Environment.OSVersion.Platform == PlatformID.Unix)
-                return "utf-8";
+                return "UTF-8";
```

Comment:
Is there a test that checks specifically for this? In my Ubuntu docker container it returns `ANSI_X3.4-1968`.